src/main/java/com/yetanalytics/xapi/model/Extensions.java

```diff
@@ -47,14 +72,23 @@ public <T> T read(String key, String jsonPathExpression, Class<T> typeKey) {
         return null;
     }
 
+    /**
+     * Remove an extension by IRI key
+     * @param key the IRI of the extension to remove
+     */
     public void remove(String key) {
         extMap.remove(key);
     }
 
+    /**
+     * @return A Set of all IRI Extension keys
+     */
```

Comment:
I think for every method that has a `@return` annotation, they should all have a description of the method itself. Otherwise when I look at the Javadocs and don't see a method description, I don't bother looking at the return values.

Reply:
So, I guess? but like they're mostly no-arg methods that JUST return a thing, so it would be nearly a duplicate of the @return line. It would in this case just say "**Returns** a Set of all IRI...". Felt redundant which is why i skipped it.

Reply:
59646edfc8cb2d3de629b961452f5aa15c75c4ac